src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -477,19 +478,23 @@ public function copyLanguage(
             $resourceLocatorStrategy = $this->resourceLocatorStrategyPool->getStrategyByWebspaceKey($webspaceKey);
         }
 
+        // The parent-document can be unknown e.g. if the requested page was the homepage
+        $parentUuid = !$parentDocument instanceof UnknownDocument ? $this->inspector->getUuid($parentDocument) : null;
```

Comment:
Wouldn't it be better to check if the `$document` is an instance of `HomeDocument`? Then we would not have to use `getParent` at all and save some time.